static/app/components/events/packageData.tsx

```diff
@@ -1,52 +1,41 @@
-import {Component} from 'react';
-
 import ClippedBox from 'sentry/components/clippedBox';
 import ErrorBoundary from 'sentry/components/errorBoundary';
 import EventDataSection from 'sentry/components/events/eventDataSection';
 import KeyValueList from 'sentry/components/events/interfaces/keyValueList';
-import {getMeta} from 'sentry/components/events/meta/metaProxy';
 import {t} from 'sentry/locale';
 import {Event} from 'sentry/types/event';
 
 type Props = {
   event: Event;
 };
 
-class EventPackageData extends Component<Props> {
-  shouldComponentUpdate(nextProps: Props) {
-    return this.props.event.id !== nextProps.event.id;
-  }
-
-  render() {
-    const {event} = this.props;
-    let longKeys: boolean, title: string;
-    const packages = Object.entries(event.packages || {}).map(([key, value]) => ({
-      key,
-      value,
-      subject: key,
-      meta: getMeta(event.packages, key),
-    }));
+export function EventPackageData({event}: Props) {
+  let longKeys: boolean, title: string;
```

Comment:
```suggestion
  let isLongKeys: boolean, title: string;
```

Reply:
its a good feedback but I don't want to change this file that much 😬